RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py

```diff
@@ -35,6 +36,7 @@
                                            #'keep ZDCDataFramesSorted_*Digis_*_*',
                                            'keep ZDCDataFramesSorted_hcalDigis_*_*',
                                            'keep ZDCDataFramesSorted_castorDigis_*_*',
+                                           'keep QIE10DataFrameHcalDataFrameContainer_hcalDigis_ZDC_*',
```

Comment:
I don't think there have been any major data flow changes from last year. Is this edit necessary then? Should I return this file to its original state, or carry over the edits further to customise_PPwithHI.py?